config/application.neon

```diff
@@ -45,3 +47,11 @@ services:
 	- TwigStan\Twig\TokenParser\DumpTypeTokenParser
 	- TwigStan\Twig\TwigFactory(environmentLoader: %twigstan.environmentLoader%)
 	- TwigStan\Twig\TwigFileNormalizer
+	- PHPStan\File\FileHelper(
+		workingDirectory: %currentWorkingDirectory%
+	)
```

Comment:
Maybe we don't need this. In most places it's handled with `Path::makeRelative`

Reply:
done